packages/unity-bootstrap-theme/vite.config.js

```diff
@@ -52,15 +49,10 @@ const c = {
       treeshake: true,
       output: {
         entryFileNames: (chunkInfo) => {
-          if (chunkInfo.name.includes('index.esm')) {
-            return 'js/bootstrap.bundle.min.[format]';
-          }
-          return "js/[name].[format]";
+          return "js/unityBootstrap.bundle.js";
```

Comment:
@davidornelas11 Would it require anything more than just reworking the output configs to allow for outputting the bundle as well as all the JS files individually? I'm thinking of situations where a consumer may be using `component-header` and so doesn't want to import that? The simple way to offer flexibility for cases like that would be to have individual as well as the bundle js files.

Reply:
Also, for consistency, what do you think about using `-`s in the filename instead of camel case?